webflow-hacks/form-hacks/2.ts

```diff
@@ -0,0 +1,42 @@
+import Cookies from 'js-cookie';
```

Comment:
Check what I told @aexshafii in his PR:
https://github.com/finsweet/fs-snippets/pull/7#discussion_r789871917
https://github.com/finsweet/fs-snippets/pull/7#discussion_r789872639

Reply:
noted